source/draw/Color.ooc

```diff
@@ -83,6 +83,7 @@ ColorYuva: cover {
 	y, u, v, a: Byte
 	normalized ::= FloatTuple4 new(this y as Float / 255, this u as Float / 255, this v as Float / 255, this a as Float / 255)
 	black ::= static This new(0, 128, 128, 255)
+	transparent ::= static This new(0, 128, 128, 0)
```

Comment:
If RGBA has `transparent` then YUVA should have it too for portability between the systems.